test/Feature/RpcSpec.hs

```diff
@@ -522,8 +523,8 @@ spec actualPgVersion =
               `shouldRespondWith`
                 [json|["hi", "there"]|]
 
-    it "returns first value for repeated params without VARIADIC" $
-      get "/rpc/sayhello?name=world&name=ignored"
+    it "returns last value for repeated params without VARIADIC" $
+      get "/rpc/sayhello?name=ignored&name=world"
```

Comment:
@wolfgangwalther I've changed this behavior. It would be a breaking change otherwise.

Reply:
Oh yes. It was my intention to not introduce a breaking change here - that's why I wrote the test. :+1: